misc/sampleapp-android-tests/tests.full.xml

```diff
@@ -294,50 +294,56 @@
         </description>
       </testcase>
     </set>
-    <set name="SampleApp-bdd" type="script">	
-      <testcase component="Sample Apps" execution_type="auto" id="hexgl_appFunction" priority="P2" purpose="Validate if sample app HexGL function could work well" status="approved" type="Functional">
+    <set name="SampleApp-bdd" type="script" ui-auto="bdd">	
+      <testcase component="Sample Apps" execution_type="manual" id="hexgl_appFunction" priority="P2" purpose="Validate if sample app HexGL function could work well" status="approved" type="Functional">
         <description>
           <pre_condition>
             1. Make sure HexGL webapp is installed
           </pre_condition>
           <post_condition />
-		  <test_script_entry test_script_expected_result="0">/opt/sampleapp-android-tests/bddrunner sampleapp-android-tests/testscripts/hexgl_appFunction.feature</test_script_entry>
+          <test_script_entry>/opt/sampleapp-android-tests/sampleapp/hexgl_appFunction.html</test_script_entry>
```

Comment:
let the "test_script_entry" and "bdd_test_script_entry" line's indentation/format consistent, same requirement for below entrys